src/gsy_e/models/market/forward.py

```diff
@@ -47,6 +57,7 @@ def __init__(self, bc: Optional[NonBlockchainInterface] = None,
         super().__init__(bc=bc, notification_listener=notification_listener,
                          readonly=readonly, grid_fee_type=grid_fee_type,
                          grid_fees=grid_fees, name=name)
+        self._open_market_slots = {}
```

Comment:
NIT, but could be helpful
```suggestion
        self._open_market_slots: Dict[DateTime, ForwardMarketSlot] = {}
```

Reply:
Fixing, thanks!